docs/configuration/rbac.md

```diff
@@ -4,252 +4,449 @@ title: RBAC
+
+Kubernetes requires user for group impersonation. That's why when a group mapping is user without `context.rbac.user` mapping defined, Botkube uses `botkube-internal-static-user` user name for impersonation.
+For example, when the following configuration is used:
 
 ```yaml
 executors:
-  "kubectl-read-only":
+  "kubectl":
     botkube/kubectl@v1:
       enabled: true
+      # ...
       context:
         rbac:
-          user:
+          # no user mapping defined
+          group:
             type: Static
             static:
-              value: kubectl-read-only
+              value: botkube-plugins-default
 ```
 
-When this executor plugin is invoked, a kubeconfig impersonating user `kubectl-read-only` is generated by Botkube
-and passed to the plugin. The plugin then can authenticate with the API server with identity of user `kubectl-read-only`.
-
-#### Example: Kubernetes source plugin with static mapping
-
-In this example a single source plugin is defined with static RBAC that maps to user `kubernetes-read-only`.
-
-1. Botkube config:
+It is equivalent to:
 
 ```yaml
-sources:
-  "kubernetes":
-    botkube/kubernetes@v1:
+executors:
+  "kubectl":
+    botkube/kubectl@v1:
+      enabled: true
+      # ...
       context:
         rbac:
           user:
             type: Static
             static:
-              value: kubernetes-read-only
+              value: botkube-internal-static-user
+          group:
+            type: Static
+            static:
+              value: botkube-plugins-default
 ```
 
-2. Kubernetes RBAC
+#### Defaults for Botkube Cloud
 
-```yaml
-  apiVersion: rbac.authorization.k8s.io/v1
-  kind: ClusterRole
-  metadata:
-    name: reader
-  rules:
-  - apiGroups: ["*"]
-    resources: ["*"]
-    verbs: ["get", "watch", "list"]
-  ---
-  apiVersion: rbac.authorization.k8s.io/v1
-  kind: ClusterRoleBinding
-  metadata:
-    name: reader
-  roleRef:
-    apiGroup: rbac.authorization.k8s.io
-    kind: ClusterRole
-    name: reader
-  subjects:
-  - kind: User
-    name: kubernetes-read-only
-    apiGroup: rbac.authorization.k8s.io
-```
+When configuring plugin on Botkube Cloud, the "Default" permissions mean that the `botkube-plugins-default` group will be used, which have read-only access to Kubernetes API and is configured during Botkube installation. See the [Default configuration](#default-configuration) section.
+
+![Cloud RBAC defaults](./assets/cloud-rbac-default.png)
+
+## Examples
 
-#### Example: kubectl executor plugin with channel name mapping
+This paragraph contains examples of RBAC configuration for different use cases.
+
+:::tip
+You can use `extraObjects` override during Botkube installation. See the [`values.yaml`](https://github.com/kubeshop/botkube/blob/v1.3.0/helm/botkube/values.yaml#L1059) for more details.
+:::
 
-In this example **kubectl** executor plugin is configured with channel name mapping and bound to two channels,
-`ch-1` and `ch-2`.
+### Use default RBAC configuration
```

Comment:
TBH I do not understand this section. Also, the padding is wrong, and the comma is missing.
![Screenshot 2023-08-22 at 12 34 50](https://github.com/kubeshop/botkube-docs/assets/17568639/e6546e12-3f37-400e-a4cd-9c86ced8fd46)

I would remove it, or we need to rephrase it. BTW above, you already have a section with:

``````markdown
executors:
  "kubectl":
    botkube/kubectl@v1:
      enabled: true
      # ...
      context:
        rbac: {} # enable kubeconfig generation
```

Here we define a plugin with _Static_ mapping to **User.rbac.authorization.k8s.io** `kubectl-read-only`.
However, such configuration will generate a kubeconfig with empty impersonation config, which effectively means an anonymous access to the Kubernetes API.
``````

Reply:
Right, I forgot to remove it as I introduced it above 🙈